[Connectors] Enable UniFi connector for Workflows (follow-up to #283915) (#283925)
elastic/kibana: docs/reference/connectors-kibana/unifi-action-type.md

You configure the console URL and an API key when creating the connector; every action then runs under that key's account and permissions.

> Every action listed below is available both as a tool for Agent Builder agents and as a sub-action on a UniFi workflow step.

Reviewer: Is this referring to the paragraph directly after, or the actions listed in the [Available actions](https://docs-v3-preview.elastic.dev/elastic/kibana/pull/283925/reference/connectors-kibana/unifi-action-type#unifi-available-actions)" section? If it's the latter, I'd recommend linking to it, like this:

```suggestion
All [UniFi Network actions](#unifi-available-actions) are available both as a tool for Agent Builder agents and as a sub-action on a UniFi workflow step.
```
